test/acceptance-tests/use-cases/test-scripts/load-workspace.js

```diff
@@ -29,20 +29,21 @@ module.exports = function() {
 
   this.When(/^I load the '(.+)' workspace from the sandbox directory$/,
   function(templateName, next) {
-    testsuite.destinationPath = testSupport.givenSandboxDir(templateName);
-    testsuite.workspace =
-      workspaceManager.getWorkspaceByFolder(testsuite.destinationPath);
     const app = require('../../../../');
     const directory = testsuite.destinationPath;
     supertest(app)
     .post('/api/Workspace/load-workspace')
-    .send(directory)
+    .send({directory: directory})
     .expect(200, function(err, response) {
       if (err) return next(err);
       const data = response.body;
       expect(data.workspaceId).to.not.to.be.undefined();
       expect(data.errors).to.not.to.be.undefined();
```

Comment:
can you assert workspaceId to equal a particular value?

data.errors check is not required because you're checking it on 42 anyways